app/views/layouts/_meta_tags.slim

```diff
@@ -0,0 +1,14 @@
+meta charset="utf-8"
+meta name="viewport" content="width=device-width,initial-scale=1"
+meta name="description" content="Jalappeno"
+meta name="author" content="Jalappeno"
+meta name="keywords" lang="en-GB" content="Let's talk about your new business ideas"
```

Comment:
can you add content for meta tags from translations? eg. `t('metatags.keywords.content')`